src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/ui/DslExtensions.kt

```diff
@@ -115,4 +123,32 @@ fun Row.textFieldWithCompletion(
     icon: Icon? = null
 ): Cell<TextFieldWithCompletion> {
     return textFieldWithCompletion(PropertyBinding(property.getter, property.setter), completions, project, icon)
+}
+
+fun Row.customTextWithBrowseButton(
```

Comment:
Let's introduce some name that better matches "text field with browse button for relative folders"